### PR TITLE
ft: Add storage quota config

### PR DESCRIPTION
**Pull request by ssalaues**
Allows for storage quota configuration to be enabled from docker entry point.

### CODE REVIEW

**Review by dora-korpar — APPROVED** [review]

**Review by giacope — APPROVED** [review]

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -116,6 +116,10 @@ if [[ "$RECORDLOG_ENABLED" ]]; then
     JQ_FILTERS_CONFIG="$JQ_FILTERS_CONFIG | .recordLog.enabled=true"
 fi
 
+if [[ "$STORAGE_QUOTA_ENABLED" ]]; then
```
Comment (bennettbuchanan) [inline docker-entrypoint.sh]: Why not set `utapi.metrics` with the metrics array value as `['location']`? That way we can extend it later for the other metric granularities.

Comment (bennettbuchanan) [reply docker-entrypoint.sh]: Seems like we may eventually want to introduce other metrics.

Comment (jonathan-gramain) [inline docker-entrypoint.sh]: Can you change the name to `STORAGE_LIMIT_ENABLED`? It is the official name of the feature and will avoid confusion with the upcoming quotas support (per user).

**Review by jonathan-gramain — CHANGES_REQUESTED** [review]

**Review by bennettbuchanan — CHANGES_REQUESTED** [review]

**Comment by jonathan-gramain** [comment]
@bert-e wait

**Comment by ssalaues** [comment]
Updated PR to accommodate the requested changes.

**Review by jonathan-gramain — APPROVED** [review]

**Review by bennettbuchanan — APPROVED** [review]

**Pull request merged**